src/app/lib/mainsail/transaction.service.ts

```diff
@@ -387,11 +382,10 @@ export class TransactionService extends Services.AbstractTransactionService {
 
 		if (input.signatory.actsWithConfirmationMnemonic()) {
 			signedTransactionBuilder = await transaction.sign(input.signatory.signingKey());
-			// transaction.secondSign(input.signatory.confirmKey());
 		}
 
 		if (input.signatory.actsWithWIF()) {
```

Comment:
maybe remove both `wif` methods as we don't import by `wif` in vault anyway and the SDK does not support it